README.md

```diff
@@ -68,15 +111,21 @@ tsci dev
 - [x] Simplified, extensible auto-routing for schematics and PCBs
 - [x] Generate footprints from text [using AI](https://text-to-footprint.tscircuit.com)
 
----
+## Community Support
+
+Have any queries? Want to reach out to the team? Have got feature ideas? Want to report any bugs? Want to discuss something? Want to attend meetings? Just want to give your feedback of using tscircuit or want to introduce yourself or want to be a part of our growing community? We got you - We primarily communicate on <a href="https://tscircuit.com/join" target="_blank">Discord Community Server</a>, of 100+ members currently, where you are a developer or not - all the primary communications happen here.
+
+## Want to contribute to tscircuit?
+
+Contributors are most welcome! We value your time and efforts in making tscircuit more awesome! ⚡💙 Please Check our <a href="https://github.com/tscircuit/snippets/blob/main/CONTRIBUTING.md" target="_blank">Contributing Guide</a> for your next contribution!
```

Comment:
We can add something like this when we have a contribution guide, and the contribution guide we're going to be extremely picky about so best left to another PR

```suggestion
```